### PR TITLE
Edit Contact Details Styling

### PR DESCRIPTION
**Pull request by mmythily**
Primary reviewer: @GPaoloni 

## Description
- Modified contact sections according to this [mock](https://marvelapp.com/prototype/ah7f313)
- Added a dedicated ContactDetailsSection.tsx
- Updated test suites in Search 

### Checklist
- [x] Corresponding issue has been opened
- [x] New tests added
- [x] Strings are localized
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Related Issues
Fixes # [CHI-1180](https://bugs.benetech.org/browse/CHI-1180)

### CODE REVIEW

**Comment by nickhurlburt** [comment]
@mmythily if this is ready for review, can you select a primary reviewer?

File: plugin-hrm-form/src/styles/search/index.tsx [hunk 1]
```diff
@@ -310,6 +310,7 @@ export const ContactDetailsIcon = icon => styled(icon)`
   color: #000000;
   width: 50px;
   height: 50px;
+  font-size: medium;
```
Comment (GPaoloni) [inline plugin-hrm-form/src/styles/search/index.tsx:313]: This is about 16px, right?

File: plugin-hrm-form/src/translations/en-US/flexUI.json [hunk 2]
```diff
@@ -280,6 +283,7 @@
   "ContactDetails-GeneralDetails-DiscussRights": "Did you discuss rights with the child?",
   "ContactDetails-GeneralDetails-SolvedProblem": "Did the child feel we solved their problem?",
   "ContactDetails-GeneralDetails-WouldRecommend": "Would the child recommend us to a friend?",
+  "ContactDetails-ActionHeaderAdded": "Added on {{date}} at {{time}} by {{counsellor}}",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/en-US/flexUI.json:286]: Nice!

**Review by GPaoloni — APPROVED** [review]
Looks good to me!

Two very small details I've spotted:
- Looks like the date format is a bit different. I'm fine with any of them but check with Jana if possible.
  - Current
    Screenshot from 2022-06-08 11-35-58
  - Mocks:
  Screenshot from 2022-06-08 11-37-35
- Looks like the updated date is not being shown for me after I edit a contact. I notice that the `updatedAt` and `updatedBy` properties are not being sent by the search endpoint, so not needed to address in this PR but is a bug that we'll need to address.

**Pull request merged**